### PR TITLE
More details for lobster documentation

### PR DESCRIPTION
**Pull request by JaGeo**
## Summary

I have added another code snippet to the documentation on how to start Lobster runs.

### CODE REVIEW

File: docs/user/codes/vasp.md [hunk 1]
```diff
@@ -258,6 +258,33 @@ VASP_CMD: <<VASP_CMD>>
...
+lobster = VaspLobsterMaker().make(structure)
+
+# update the incar
+lobster = update_user_incar_settings(lobster, {"NPAR": 4})
+# run the job
+run_locally(lobster, create_folders=True, store=store)
+```
+
+
+It is, however,  computationally very beneficial to define two different types of job scripts for the VASP and Lobster runs.
```
Comment (janosh) [inline docs/user/codes/vasp.md:287]: Not clear to me what is meant. Is it because you have double the max wall time with 2 jobs?

Comment (JaGeo) [reply docs/user/codes/vasp.md:287]: Two types of parallelisations. MPI vs. OpenMP

Comment (JaGeo) [reply docs/user/codes/vasp.md:287]: Actually something we could also fix in custodian. Currently you cannot change number of openmp threads as part of the VASP or Lobster command.

Comment (JaGeo) [reply docs/user/codes/vasp.md]: @janosh tried to add the explanation. Cleaner now?

Comment (janosh) [reply docs/user/codes/vasp.md]: Perfect!

**Comment by janosh** [comment]
Thanks @JaGeo! 👍

**Pull request merged**